src/masternodes/mn_checks.cpp

```diff
@@ -3855,13 +3855,11 @@ class CCustomTxApplyVisitor : public CCustomTxVisitor {
         if (obj.evmTx.size() > static_cast<size_t>(EVM_TX_SIZE))
             return Res::Err("evm tx size too large");
 
-        if (!evm_validate_raw_tx(HexStr(obj.evmTx))) {
+        if (!evm_prevalidate_raw_tx(HexStr(obj.evmTx)))
             return Res::Err("evm tx failed to validate");
-        }
 
-        if (!evm_queue_tx(evmContext, HexStr(obj.evmTx))) {
+        if (!evm_queue_tx(evmContext, HexStr(obj.evmTx), tx.GetHash().ToArrayReversed()))
```

Comment:
Seems like the changes in commit ba4eac0622 (https://github.com/DeFiCh/ain/pull/2025/commits/ba4eac0622ccffadf100b01d39c62472a09c3212) in PR #2025 is not reflected here.

Reply:
Good spot. I'm updating the CPP changes and will look for this.

Reply:
This is now included. It was merged after this PR was created so was missing initially.